### PR TITLE
Server/db

### PR DESCRIPTION
**Pull request by mechina-19**
Code is good, nothing to see here!

### CODE REVIEW

File: server/.gitignore
Comment (number10mechina) [inline server/.gitignore:1]: great

File: server/db.py [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+       # Insert data into passengers table
+        self.cursor.execute("INSERT INTO passengers (username, password) VALUES (?, ?)", ("user1", hashlib.sha256("passwod123".encode()).hexdigest()))
+
+        # Insert data into drivers table
+        self.cursor.execute(
+            """
+            INSERT INTO drivers (username, password, driver_id, name, license_plate)
+            VALUES 
+                ('driver01', '""" +  hashlib.sha256("passwod123".encode()).hexdigest() +"""', '6151181', 'Alice Johnson', 'ABC1234'),
+                ('driver02', '""" +  hashlib.sha256("passwod123".encode()).hexdigest()+ """', '1455184', 'Chris Lee', 'XYZ5678'),
```
Comment (number10mechina) [inline server/db.py:76]: add a function that adds the a driver to the db. you shouldn't initiate the db with those values as a solution

File: server/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+fastapi
+pydantic
```
Comment (number10mechina) [inline server/requirements.txt]: add the rest of the used packeges

File: server/server.py [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+        raise HTTPException(status_code=401, detail=str(e))
+
+
+@app.get("/update-arrival-time/{station_id, bus_id}")
+def update_arrival_time(station_id: int, bus_id: int):
+    """
+    Get real-time bus location and arrival time to a given station
+    """
+
+    return {"Arriving": "12:54:32", "Location": {"Lat": "33.4234242", "Lon": "34.334211"}}
```
Comment (number10mechina) [inline server/server.py]: add implementation in next version

File: server/db.py (hunk 1 above)
Comment (ozhanich) [reply server/db.py:76]: added a function that adds drivers and changed the initializations.

File: server/requirements.txt (hunk 2 above)
Comment (ozhanich) [reply server/requirements.txt]: added uvicorn

File: server/server.py (hunk 3 above)
Comment (ozhanich) [reply server/server.py]: ok

File: server/db.py [hunk 4]
```diff
@@ -107,6 +97,31 @@ def initialize_db(self, zip_path):
...
+                INSERT INTO drivers (username, password, driver_id, name, license_plate)
+                VALUES (?, ?, ?, ?, ?)
+            """, (username, hashed_password, driver_id, name, license_plate))
+            self.connection.commit()
+            return True
+        except sqlite3.IntegrityError:
+            return False
+        
     def close(self):
         self.connection.close()
```
Comment (number10mechina) [inline server/db.py:127]: great work mate!

**Pull request merged**